src/com/sap/piper/versioning/DockerArtifactVersioning.groovy

```diff
@@ -56,8 +56,8 @@ class DockerArtifactVersioning extends ArtifactVersioning {
         def version = null
         for (def i = 0; i < lines.size(); i++) {
             if (lines[i].startsWith('FROM') && lines[i].indexOf(':') > 0) {
-                version = lines[i].split(':')[1]
-                break
+                def imageParts = lines[i].split(':')
+                version = imageParts[imageParts.size()-1]
```

Comment:
Up to now we used the first `FROM` statement for resolving the version.
Now we use the latest `FROM` statement for resolving the version.

It that change intended? Is the latest `FROM` statement always the right one?

Reply:
Good catch! added `break` again